src/e2e/specs/dashboard.spec.ts

```diff
@@ -642,3 +718,42 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Navigation`, (
     );
   });
 });
+
+test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Data Breaches`, () => {
+  test.beforeEach(async ({ landingPage, page, authPage }) => {
+    await forceLoginAs(
+      "joe@mailinator.com",
```

Comment:
We usually don't put credentials out in plain text, especially passwords. Especially here you can reuse the var for password.
Even though it's for test cases, it's bad security practices